docusaurus/docs/operate/quickstart/supplier_cheatsheet.md

```diff
@@ -0,0 +1,265 @@
+
+:::
+
+Retrieve your external IP address:
+
+```bash
+EXTERNAL_IP=$(curl -4 ifconfig.me/ip)
+```
+
+Choose a port that'll be publicly accessible from the internet (e.g. `8545`)
+
+```bash
+sudo ufw allow 8545/tcp
+```
+
+Create a Supplier stake configuration file:
+
+```bash
+cat <<EOF > /tmp/stake_supplier_config.yaml
+owner_address: $SUPPLIER_ADDR
+operator_address: $SUPPLIER_ADDR
+stake_amount: 1000069upokt
+default_rev_share_percent:
+  $SUPPLIER_ADDR: 100
+services:
+  - service_id: "morse"
+    endpoints:
+      - publicly_exposed_url: http://$EXTERNAL_IP:8545
+        rpc_type: JSON_RPC
+EOF
+```
+
+And run the following command to stake the `Supplier`:
+
+```bash
+poktrolld tx supplier stake-supplier --config /tmp/stake_supplier_config.yaml --from=$SUPPLIER_ADDR $TX_PARAM_FLAGS $NODE_FLAGS
+
+# Optionally, to avoid entering the password each time:
+# poktrolld tx supplier stake-supplier --config /tmp/stake_supplier_config.yaml --from=$SUPPLIER_ADDR $TX_PARAM_FLAGS $NODE_FLAGS --keyring-backend test
+```
+
+After about a minute, you can check the `Supplier`'s status like so:
+
+```bash
+poktrolld query supplier show-supplier $SUPPLIER_ADDR $NODE_FLAGS
+```
+
+## RelayMiner Configuration
+
+### Configure the RelayMiner
+
+```bash
+cat <<EOF > /tmp/relayminer_config.yaml
+default_signing_key_names:
+  - supplier
+smt_store_path: /home/pocket/.poktroll/smt
+pocket_node:
+  query_node_rpc_url: https://shannon-testnet-grove-rpc.beta.poktroll.com
+  query_node_grpc_url: https://shannon-testnet-grove-grpc.beta.poktroll.com:443
+  tx_node_rpc_url: https://shannon-testnet-grove-rpc.beta.poktroll.com
+suppliers:
+  - service_id: "morse"
+    service_config:
+      backend_url: "https://pocket-rpc.liquify.com"
+      publicly_exposed_endpoints:
+        - $EXTERNAL_IP
+    listen_url: http://0.0.0.0:8545
+metrics:
+  enabled: false
+  addr: :9090
+pprof:
+  enabled: false
+  addr: :6060
+EOF
+```
+
+### Start the RelayMiner
+
+```bash
+poktrolld \
+    relayminer \
+    --grpc-insecure=false \
+    --log_level=debug \
+    --config=/tmp/relayminer_config.yaml \
+    # --keyring-backend=test
+```
+
+### Secure vs Non-Secure `query_node_grpc_url`
+
+In `/tmp/relayminer_config.yaml`, you'll see that we specify an endpoint with TLS
+for `query_node_grpc_url`.
+
+If `grpc-insecure=true` then it **MUST** be an HTTP port, no TLS.
+
+The Grove team exposed one such endpoint on one of our validators for Beta Testnet at `http://149.28.34.68:9090`.
+It can be validated with `grpcurl -plaintext 149.28.34.68:9090`; note that the `-plaintext` flag meaning no TLS encryption.
+
+If `grpc-insecure=false`, then it **MUST** be an HTTPS port, with TLS.
+
+The Grove team exposed one such endpoint on one of our validators for Beta Testnet at `https://shannon-testnet-grove-grpc.beta.poktroll.com:443`.
+It can be validated with `grpcurl https://shannon-testnet-grove-grpc.beta.poktroll.com:443`; note no `-plaintext` flag meaning no TLS encryption.
```

Comment:
```suggestion
It can be validated with `grpcurl https://shannon-testnet-grove-grpc.beta.poktroll.com:443`; note the absence of the `-plaintext` flag meaning no TLS encryption.
```